index.html

```diff
@@ -76,7 +75,6 @@ <h3>I am a Software Developer</h3>
         </div>
       </section>
 
-
       <!-- work Section -->
       <div class="work" id="portfolio">
         <div id="work-top">
```

Comment:
- [ ] As already pointed out by the previous reviewer, there should be a popup which is created dynamically for each project.
- [ ] Also, please ensure that the projects section is created dynamically. This will make it easy for you to add projects in the future.
- [ ] You can go ahead and add the `See Project` button for every card.